data/server_locations.json

```diff
@@ -66952,11 +66952,11 @@
                 "status": "unvisited",
                 "external_url": "null",
                 "internal_url": "null",
-                "latitude": 42.523759313620914,
-                "longitude": -87.89325782771036,
+                "latitude": "42.523759313620886",
+                "longitude": "-87.89325782771036",
```

Comment:
@jannisborn was it always the case that lat and lon are sometimes floats and sometimes strings?? Seems to be a mix here. Maybe due to merge of all_locations and server_locations?